src/Shared/AlertMessage/AlertMessage.tsx

```diff
@@ -0,0 +1,16 @@
+interface Props {
+  message: string;
+}
+export default function AlertMessage(props: Props) {
```

Comment:
I suggest naming this component as `ErrorMessage`, as this message is shown when errors appear in the form not an alert.

Reply:
You can also pass the message between the component and it will be provided to the children prop.